about/team/index.md

```diff
@@ -2,61 +2,109 @@
+
+* Jouk Jansen maintains wxWidgets for OpenVMS systems.
+
+* Danny Scott and Xavier Perrissoud build wxMSW binaries for wxWidgets
+  releases.
+
+* Special thanks to Eric Jensen for his always pertinent contributions to the
+  various mailing list discussions.
+
+* Many more people ought to be listed here but for now just a few more
+  names and thanks to: AliKet, Andreas Falkenhahn, Daniel Kulp, Ian McInerney,
+  Paul Kulchenko, Konstantin S. Matveyev, Lauri Nurmi, New Pagodi, Cătălin
+  Răceanu, Randalphwa, Dimitri Schoolwerth, Pavel Tyunin... And sorry in
+  advance for anybody forgotten here (and please see the top of this page).
+
+
+## Core Team Alumni
+
+Past core team members not actively involved in wxWidgets development any
+longer, but without whom wxWidgets wouldn't have been possible:
+
 * [Dr. Julian Smart](mailto:julian@anthemion.co.uk) - Julian Smart started the
-  wxWidgets project at the University of Edinburgh in 1992. He works on various
+  wxWidgets project at the University of Edinburgh in 1992. He worked on various
   aspects of the project, including wxX11, wxWinCE, release engineering, and
   web site maintenance. Julian's company
   [Anthemion Software Ltd.](http://www.anthemion.co.uk/) produces consumer and
   developer products, including the story development software
   [StoryLines](http://www.writerscafe.co.uk/), and the wxWidgets dialog editor
   [DialogBlocks](http://www.anthemion.co.uk/dialogblocks).
 
-* [Václav Slavík](mailto:vaclav.slavik@matfyz.cz) - Václav Slavík works on wxHTML, the
-  XRC system, the new build system, wxMGL, and other areas. He also works on a
-  variety of other [open source projects](http://sourceforge.net/users/vaclavslavik/).
-
-* [Robin Dunn](mailto:robin@alldunn.com) - Robin works on general wxWidgets
-  maintenance as well as the widely used [wxPython](http://wxpython.org/)
-  system. He is also working for a major wxPython user, the
-  [Open Source Applications Foundation](http://www.osafoundation.org/), headed
-  up by industry leader Mitch Kapor.
-
+* [Dr. Robert Roebling](mailto:robert@roebling.de) - Robert Roebling has
+  written the initial GTK port and has overseen its development since then.
 
-## Major Contributors
 
-This list is mainly based on commit access but there are of course many more
-people who make important contributions to wxWidgets.
+## Other Historical Contributors
 
-* [Bryan Petty](mailto:bryan@ibaku.net) - Bryan contributed the refactoring of
-  the wxVScrolledWindow for the additions of the wxHScrolledWindow and
-  wxHVScrolledWindow, as well as several patches to the wxAUI library. He also
-  worked with Francesco Montorsi on the conversion of the reference manual from
-  LaTeX to Doxygen, designed the wxWidgets logo, and hosts this website, the
-  wxWiki, and the online reference manuals.
+Incomplete list of other people who made important contributions to wxWidgets
+in the past but are not actively involved into development any longer:
```

Comment:
A lot of the past contributors texts remain in present tense, which reads a bit weird, given the new context for them (and me :smile: )

Reply:
I thought about this, but the problem is that if we start changing the tense, we should probably also update other parts that are not really relevant any more and I was too lazy to do it. And still am, so I've just added an explanatory note about this for now.

BTW, should I move you to the "still active" part? Please let me know...

Reply:
Past contributor is fine. (currently backlogged) maintaining in downstream doesn't really matter here imo, or all downstream maintainers should be listed as contributors or something.